rtb/analisi-requisiti/src/casi-uso.d/cliente-ristoratore-chat.tex

```diff
@@ -1 +1,55 @@
-B
+
+\textbf{UC49-Avvia chat}
+\begin{itemize}
+\item \textbf{Attore principale:} Cliente.
+\item \textbf{Precondizioni:} È in corso una prenotazione e il cliente non ha ancora avviato la chat. % TODO non so come mettere a parole che deve essere il cliente a voler comunicare con il ristoratore
```

Comment:
Metterei come precondizione solo che il cliente è in fase di prenotazione

Reply:
In realtà il cliente può avviare una chat anche senza il bisogno che ci sia una prenotazione.
Quindi sarebbe a parer mio: "l'utente è nella pagina di visualizzazione ristorante (uc18)"